### ISSUE TITLE
Slightly off-resolution files grouped separately

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dizzydezz/jmm#352]
```
What steps will reproduce the problem?
1. Have video file that has 1274x720 dimensions.
2.
3.

What is the expected output? What do you see instead?
Expected: Under "Collection > [anime] > Series Info tab > More Information 
section > File Summary dropdown" to have all the HD episodes by 1 fansub, 
grouped.
Saw: An episode who's dimensions are 1274x720 is grouped separately as non-HD.

What version of the product are you using? On what operating system?
3.1.23.0

Please provide any additional information below.
On AniDB, Fairy Tail episode 5 released by Kyuubi (which is 1274x720) is listed 
as HD, however, JMM separates it out of the main grouping.
Is it possible to make JMM group a range of resolutions under a standard 
resolution, since some groups can make the files a bit off-sized, eg, 1270-1290 
x 710-730, under 1280x720.
```

Original issue reported on code.google.com by `inferk...@gmail.com` on 8 Dec 2012 at 3:07

**Comment by GoogleCodeExporter** [comment]
```
This is also evident with [DB] Naruto Shippuuden episodes 54-62, 64-65 being 
856x480 and [DB] Naruto Shippuuden 63, 66-91 being 848x480 & thus being grouped 
separately.
```

Original comment by `inferk...@gmail.com` on 10 Dec 2012 at 11:13

**Comment by GoogleCodeExporter** [comment]
```
Ok good idea, I'll probably put in a 10% variation to the normal check.
Which should cover most scenarios
```

Original comment by `werndly...@gmail.com` on 10 Dec 2012 at 11:15
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 3.1.26
```

Original comment by `werndly...@gmail.com` on 18 Dec 2012 at 2:05
- Changed state: **Fixed**
